src/accounts/accountManager.ts

```diff
@@ -122,35 +133,45 @@ export class AccountManager {
 		// newaccount alone is enough. If there is a system contract with the buyrambytes action,
 		// then we definitely need to do it, but if there isn't, then trying to call it is an error.
 		if (systemContract.actions.has('buyrambytes')) {
+			const bytesToBuy = options?.bytesToBuy ?? 4000000;
+			logMessage += ` with ram: ${bytesToBuy}`;
+
 			actions.push({
 				account: 'eosio',
 				name: 'buyrambytes',
 				authorization: creator.active,
 				data: {
 					payer: creator.name,
-					receiver: account,
-					bytes: 8192,
+					receiver: account.name,
+					bytes: bytesToBuy,
 				},
 			});
 		}
 		// Same deal for delegatebw. Only if it's actually a thing.
 		if (systemContract.actions.has('delegatebw')) {
+			const delegateAmount = '10.0000 EOS';
```

Comment:
From memory trying to use 10 EOS was breaking with our test chain setup, which is why we were using 10 SYS. Have the later versions of EOS changed this? (e.g. Have you tested that this definitely works? If so, that's all I need, just wanted to check because I remember it causing issues before.)

Reply:
This worked for me. I was ususally seeing issues where the system contract would not actually get set which is why had delays and retries to get it to work. It also fail silently when it didn't get set which made it even worse to bugfix.

Reply:
Cool, as long as it's working for you I'm good. I think the SYS token is the default if you don't have some of the system contracts set up (which I think you now have set up), so the more we can be like mainnet the better in my opinion. 👍